### PR TITLE
ADSB server support

### PR DESCRIPTION
**Pull request by DonLakeFlyer**
Configuration example is USB SDR dongle, then run ```dump1090 --net``` tool to provide SBS messages over 127.0.0.1:30004. Default config settings are for this setup. This will work for any server which you can connect to over TCP which provides SBS stream of information.

Configurable from General Settings page:
Screen Shot 2019-11-15 at 7 19 41 PM

### CODE REVIEW

**Comment by Antiheavy** [comment]
is it dump190 or dump1090?

**Comment by DonLakeFlyer** [comment]
> is it dump190 or dump1090?

Oops, right. dump1090

**Comment by Antiheavy** [comment]
Is QGC able to automate running the dump1090 application if a properly configured USB SDR is detected?

**Comment by DonLakeFlyer** [comment]
No, you have to set that up yourself. QGC just supports connecting to an ADSB server outputing SBS messages over a TCP connection. That could be an SDR+dump1090 combination or it could be something else.

File: ChangeLog.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Note: This file only contains high level features or important fixes.
 
 ### 3.6.0 - Daily Build
 
+* ADSB: Added support for connecting to SBS server. Adds support for ADSB data from USB SDR Dongle running 'dump190 --net' for example.
```
Comment (Antiheavy) [inline ChangeLog.md]: should this read: ".... running 'dump1090 --net' for example." ?

**Comment by DonLakeFlyer** [comment]
yup

File: ChangeLog.md (hunk 1 above)
Comment (DonLakeFlyer) [reply ChangeLog.md]: fixed

**Pull request merged**